spec/models/article_spec.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Article, type: :model do
+  it 'returns title' do
+    article = Article.create(
+      title: 'First Article',
+      text: 'Some text for First Article',
+      user_id: 1,
+      status: 'published',
+      language: 'eng'
+    )
+    expect(article.title).to eq('First Article')
+  end
+
+  it 'returns text' do
+    article = Article.create(
+      title: 'First Article',
+      text: 'Some text for First Article',
+      user_id: 1,
+      status: 'published',
+      language: 'eng'
+    )
+    expect(article.text).to eq('Some text for First Article')
+  end
+
+  it 'returns status' do
+    article = Article.create(
```

Comment:
add factory_bot + faker gems

CC @mmrshk

Reply:
I added these gems. And I will rebuild the rspec tests.